src/daqlite/CustomTofPlot.cpp

```diff
@@ -112,3 +118,17 @@ void CustomTofPlot::clearDetectorImage() {
   addData(HistogramTofData);
   plotDetectorImage(true);
 }
+
+// MouseOver, display coordinate and data in tooltip
+void CustomTofPlot::showPointToolTip(QMouseEvent *event) {
+  int x = this->xAxis->pixelToCoord(event->pos().x());
+
+  // Calculate coordinates grouped with bins and retrive data
+  // according to the identified bin.
+  int stepSize = int(mConfig.TOF.MaxValue / mConfig.TOF.BinSize);
+  int dataBinPos = int((x - stepSize / 2) / stepSize);
+  int tofBinPos = int((x + stepSize / 2) / stepSize) * stepSize;
```

Comment:
This is then the TOF value, right?
tofValue = ....

But why a '+' here and not '-' like in the bin calculation?

Reply:
It's quite complicate to explain but I had to use different calculation from the coordinate I getting from the representation layer to the data index. So in case of data I had to calculate the lower edge of the graphical column for the proper TOF value which should be in the middle of the graphical column I had to calculate the higher edge.